core/src/main/resources/org/kohsuke/stapler/bind.js

```diff
@@ -1,6 +1,6 @@
 // bind tag takes care of the dependency as an adjunct
 
-function makeStaplerProxy(url,crumb,methods) {
+function makeStaplerProxy(url,staplerCrumb,methods) {
```

Comment:
Renaming this to avoid a clash with the Jenkins global variable named `crumb`, which is needed to set the "Jenkins-Crumb" header later below (not to be confused with the "Crumb" header, which is what this second argument is for…).